apps/governance/src/i18n/translations/dev.json

```diff
@@ -159,6 +159,7 @@
   "Tokens are held in different <trancheLink>Tranches</trancheLink>. Each tranche has its own schedule for how the tokens are unlocked.": "Tokens are held in different <trancheLink>Tranches</trancheLink>. Each tranche has its own schedule for how the tokens are unlocked.",
   "proposals": "Proposals",
   "proposal": "Proposal",
+  "PROPOSAL": "PROPOSAL",
```

Comment:
question (non-blocking): seems weird at first to see the same piece of text with different casing in the translations dictionary. if this is purely for styling, i'd probably expect this to be done via a tailwind `uppercase` class where needed, or using `toUpperCase` if that's not possible. not sure

Reply:
Yes, that's a fair point. Thanks